shared/lib_battery_dispatch.cpp

```diff
@@ -774,7 +774,6 @@ void dispatch_automatic_t::dispatch(size_t year,
 
 bool dispatch_automatic_t::check_constraints(double &I, size_t count)
 {
-    m_batteryPowerFlow->calculate();
```

Comment:
Why does the batteryPowerFlow no longer have to be calculated when calling the constraint check? Is the power flow ran outside of the call?

Reply:
Right, I'd added this line here during debugging a few commits ago and forgot to remove it actually. But yup, the power flow calculation is run by `battery:run` before `check_constraints` is run.